buildconfig/makeref.py

```diff
@@ -14,7 +14,9 @@
 
 def Run():
     try:
-        return subprocess.run(['sphinx-build',
+        path_to_sphinx_build = os.path.join(os.path.dirname(sys.executable), "Scripts", "sphinx-build")
```

Comment:
This isn't x-platform. 

Maybe it's worth making this happy by default on windows too. It could be changed to first look in the Scripts folder, and if there use that - otherwise use "sphinx-build" as before. Or can sphinx-build be called as a python module with -m?

Reply:
Thanks for the comment. I was not sure if this would work on linux and similar platforms. The problem that I encountered was that this subprocess used a sphinx-build from another python installation but not the one I war running setup.py

I could try to check if exists and fallback to the way as it was before, but it I would prefer a more elegant/portable solution. I'll look into the -m options of sphinx.

Reply:
@illume updated to use the module directly

Reply:
Thanks @dr0id